app/controllers/signin.js

```diff
@@ -0,0 +1,9 @@
+const signinService = require('../services/signin');
+
+class SigninController {
+  async authenticate(req, res, next) {
+    res.json(await signinService.authenticate(req.body));
```

Comment:
Please use an envelope like this `{ token: await signinService.authenticate(req.body) }`.
 
Also good to know: https://blog.mwaysolutions.com/2014/06/05/10-best-practices-for-better-restful-api/